digdag-standards/src/main/java/io/digdag/standards/operator/td/TDClientFactory.java

```diff
@@ -8,19 +9,27 @@
 {
     static TDClient clientFromConfig(Config params)
     {
+        return clientFromConfig(params, false);
+    }
+
+    static TDClient clientFromConfig(Config params, boolean retries)
+    {
 
         String apikey = params.get("apikey", String.class).trim();
         if (apikey.isEmpty()) {
             throw new ConfigException("Parameter 'apikey' is empty");
         }
 
-        TDClient client = TDClient.newBuilder(false)
+        AbstractTDClientBuilder<TDClient> builder = TDClient.newBuilder(false)
```

Comment:
Ah. Turns out that the setter methods return `AbstractTDClientBuilder`. That's pretty broken.